editor/plugins/mesh_library_editor_plugin.cpp

```diff
@@ -136,9 +136,11 @@ void MeshLibraryEditor::_import_scene(Node *p_scene, Ref<MeshLibrary> p_library,
 					continue;
 				}
 
-				//Transform3D shape_transform = sb->shape_owner_get_transform(E);
-
-				//shape_transform.set_origin(shape_transform.get_origin() - phys_offset);
+				Transform3D shape_transform;
+				if (p_apply_xforms) {
+					shape_transform = mi->get_transform();
+				}
+				shape_transform *= sb->get_transform() * sb->shape_owner_get_transform(E);
```

Comment:
If we don't get into the "if" statement, are you not doing a multiplication with an uninitialised variable?

Reply:
It is not uninitialized: a `Transform3D` is initialized to the identity transform, i.e. with identity basis matrix and zero origin.

Reply:
Alright didn't know, thanks!